examples/trade/policy/ppo.py

```diff
@@ -0,0 +1,255 @@
+import torch
+
+import numpy as np
+from torch import nn
+import torch.nn.functional as F
+from typing import Dict, List, Tuple, Union, Optional
+
+from tianshou.policy import PGPolicy
+from tianshou.data import Batch, ReplayBuffer
+from tianshou.data import to_torch
+from numba import njit
+import sys
+
+sys.path.append("..")
```

Comment:
Such behaviour is not encouraged.
 You can use some tricks like this to organize your package, 

https://github.com/amontalenti/elements-of-python-style#avoid-syspath-hacks